modules/Kconfig.mbedtls

```diff
@@ -3,8 +3,17 @@
 # Copyright (c) 2016 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 
+config MBEDTLS_PROMPTLESS
+	bool
+	help
+	  Symbol to disable the prompt for MBEDTLS selection.
```

Comment:
Branding: MBEDTLS => mbed TLS, mbedTLS => mbed TLS. 

Also. there is a duplication of terms that seems a bit odd:  mbedTLS and MBEDTLS in same sentence

Reply:
No, because this `MBEDTLS` refers to the Kconfig symbol `MBEDTLS`, and that symbol is upper case.

https://github.com/nrfconnect/sdk-zephyr/blob/7de47f00d4e40491abb483a9dc2d67a5d4a685d0/modules/Kconfig.mbedtls#L6